jetty-ee10/jetty-ee10-servlet/src/test/java/org/eclipse/jetty/ee10/servlet/DefaultServletTest.java

```diff
@@ -465,13 +465,13 @@ public void testListingFilenamesOnlyUrlResource() throws Exception
 
         assertThat(response.getStatus(), is(HttpStatus.OK_200));
         body = response.getContent();
-        assertThat(body, containsString(">#hashcode&nbsp;<")); // text on page
+        assertThat(body, containsString(">#hashcode&#160;<")); // text on page
```

Comment:
why is the numeric value better than the slightly descriptive `nbsp`?

Reply:
`&nbsp;` is an HTML named entity that isn't valid on XHTML.

Well ... technically it's not valid on XML.
XML only had 5 named entities (`&lt;`, `&gt;`, `&amp;`, `&apos;` and `&quot;`).
I'll see if I can find a valid xsd/dtd that could possibly bring in more named entities and remain standards supported.

Reply:
I added the DTD for xhtml and have a testcase that validates against the DTD now.
I have restored the `&nbsp;` usage.